lib/util.js

```diff
@@ -1481,3 +1481,24 @@ exports.getScrollBarWidth = function () {
 
   return (w1 - w2);
 };
+
+exports.topMost = function (pile, accessors) {
```

Comment:
@eymiha It looks like this PR broke the develop branch. Can you please checkout the current develop branch and test some simple network examples. It looks like the funktion `topMost` needs to be fixed.